### PR TITLE
improve code climate 

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Brief feature description
- files changes to improve code climate grades. The files that were changed are **search_grid**, **search_input** and **search_list** from search pack in react.

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
-no

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

File: app/javascript/search/search_input.js [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
-    this.setState({
-      facilityTypeId: selectedFacilityType.id
-    }
-    )
-  }
+
   sendSearchInput (event) {
     event.preventDefault()
-    const finalQuery = (this.state.countyId + ',' + this.state.facilityTypeId + ',' + this.refs.facilityId.value + ',' + this.refs.facilityHome.value + ',' + this.refs.facilityAddress.value)
+    const finalQuery = (this.state.searchId.countyId + ',' + this.state.searchId.facilityTypeId + ',' + this.refs.facilityId.value + ',' + this.refs.facilityName.value + ',' + this.refs.facilityAddress.value)
```
Comment (ratneshraval) [inline app/javascript/search/search_input.js]: I didn't know we were using refs, you don't have to fix it here but we need to get rid of it.

**Comment by Nasir2712** [comment]
retest this please

**Comment by anuroop15** [comment]
Jenkins retest this please.

File: app/javascript/facility/complaints.jsx [hunk 2]
```diff
@@ -8,21 +8,19 @@ export default class Complaints extends React.Component {
...
-            <td data-label='assigned worker'> {complaint.assigned_worker} </td>
-            <td data-label='control number' > {complaint.control_number} </td>
-            <td data-label='priority level'> {complaint.priority_level} </td>
-            <td data-label='status'> {complaint.status} </td>
-            <td data-label='approval date'> {complaint.approval_date} </td>
-          </tr>
-        )
-      })
-    }
+    facilityComplaintsTable = complaints.map((complaint) => {
```
Comment (anuroop15) [inline app/javascript/facility/complaints.jsx]: how complaints are handled?? if FacilityComplaints doesn't have complaints

**Pull request merged**